### DISCUSSION TITLE
Extremely large fonts

### DISCUSSION THREAD

**Discussion by bahronheim** [micropython/micropython#10701] [Using MicroPython] [answered]
I have a project that I'm working on that uses a Waveshare 4.2" epaper display with a Pi Pico. The goal for this project is for me to be able to read a few bits of info from across the screen, but since I'm very new to Micropython (and Python itself, if I'm honest) I'm having some trouble figuring out how to get the text large enough.
I've tried using Peter Hinch's Writer Class (https://github.com/peterhinch/micropython-font-to-py/blob/master/writer/WRITER.md), and I got it working, but even with a size 64 font it's not big enough. I'd like the fonts to be about twice this big. I don't have enough memory on the Pico for an 80-pt font file, and a 96-pt file wouldn't even generate with the font-to-py utility!
I wanted to see if anyone had good ideas for how to accomplish my goals (i.e. if there was a way to use a smaller font and then scale it, etc. ), or if I should learn how to freeze the font module for the 80-pt and see if that is readable.
I'd appreciate any help!

**Comment by peterhinch** [comment, marked as answer]
The fonts produced by [font_to_py](https://github.com/peterhinch/micropython-font-to-py) are optimised for freezing: if you freeze them, their memory usage is minimal because they are stored in Flash. I suggest you learn the technique.

**Reply by bahronheim** [reply]
Thank you for the reply! I actually tried building my first firmware yesterday and it seemed to GP smoothly. I'm going to try it out today to see if I can at least get the 80 point font running. If that works, I might see if I can figure out how to keep font-to-py from overflowing on the 96-pt version. 
Thank you so much for the help and for your amazing work! I've learned a ton just by reading your previous posts!
I just wanted to see if I was missing a better solution by taking a brute force route of just using bigger numbers. It felt like there could be an easier way that I was just missing from inexperience.

**Comment by rami6711** [comment]
I have sizable font: https://github.com/rami6711/fb_plus
